int32 timestamp
uint32 seq

#indicate the result of the yaw observation routine
uint32 code 

float32 roll
float32 pitch

float32 baselineY_IMU
float32 baselineY_Filter
float32 navYaw
float32 observedYaw

AutoNav/filter_state prior_state

AutoNav/filter_state posterior_state